cmd/cluster/support/delete.go

```diff
@@ -117,10 +117,10 @@ func (o *deleteOptions) run() error {
 	/*conditions to check the presence of --all & -i flags;
 	also, checking if there is one or more limited support resonID before deleting the same */
 	var limitedSupportReasonIds []string
-	limitedSupportReasons, err := getLimitedSupportReasons(o.clusterID)
+	limitedSupportReasons, _, err := getLimitedSupportReasons(o.clusterID)
 
 	if len(limitedSupportReasons) == 0 {
-		return fmt.Errorf("Cluster is not in limited support. \n")
+		return fmt.Errorf("No limited support reasons found or all reasons are overridden, the cluster is fully supported \n")
```

Comment:
Is that accurate here? I think here were using the _actual_ count of limited support reasons, so it will include overridden ones, right?